packages/polymarket-notifier/src/polymarketNotifier.js

```diff
@@ -170,22 +176,30 @@ class PolymarketNotifier {
         question
         outcomes
         outcomePrices
+        liquidityNum
```

Comment:
Note: this is unrelated to this PR and something we could address in a follow-up.

Thoughts on removing the check that this question is resolved by specific addresses, binaryAdapterAddress and binaryAdapterAddress?

I just say this in the interest of making this less likely to fail silently in the case of an upgrade to a new resolver type on polymarket's side.

Could we, instead, just include a simple ABI (I think we only call common methods on each of these contracts) that has the common methods and use it for all resolution contracts? If we encounter a contract that doesn't match that ABI, then this could error and page the on-call so we know something changed.